oracle/config/app.go

```diff
@@ -74,11 +100,23 @@ func (c *AppConfig) ValidateBasic() error {
 	}
 
 	if len(c.OracleAddress) == 0 {
-		return fmt.Errorf("oracle address must not be empty")
+		return fmt.Errorf("poorly formatted app.toml (oracle subsection): oracle address must not be empty")
 	}
 
 	if c.ClientTimeout <= 0 {
-		return fmt.Errorf("oracle client timeout must be greater than 0")
+		return fmt.Errorf("poorly formatted app.toml (oracle subsection): oracle client timeout must be greater than 0")
+	}
+
+	if c.MaxAge <= 0 || c.MaxAge > MaxMaxAge {
+		return fmt.Errorf("poorly formatted app.toml (oracle subsection): oracle max age must be between 0 and %s", MaxMaxAge)
+	}
+
+	if c.Interval <= 0 || c.Interval > MaxInterval {
+		return fmt.Errorf("poorly formatted app.toml (oracle subsection): oracle interval must be between 0 and %s", MaxInterval)
+	}
+
+	if c.Interval >= c.MaxAge {
+		return fmt.Errorf("poorly formatted app.toml (oracle subsection): oracle interval must be strictly less than max age")
```

Comment:
I think it's good to have these validations, but we should be setting defaults for people. Nobody should really be touching these on their own.